apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Auth0PostLoginApi } from '@vedicium/auth0-actions-sdk';
+import nock from 'nock';
+import { onExecutePostLogin } from '../post-login-connect-user';
+import type {
+  Auth0PostLoginEventWithSecrets,
+  DeepPartial,
+  User,
+} from '../types';
+
+const user: User = {
+  created_at: '2020-08-17T14:01:53.691Z',
+  email: 'joao.tiago@yld.io',
+  email_verified: true,
+  family_name: 'Tiago',
+  given_name: 'Joao',
+  identities: [
+    {
+      provider: 'google-oauth2',
+      access_token: 'accessToken',
+      expires_in: 3599,
+      user_id: 'userId',
+      connection: 'google-oauth2',
+      isSocial: true,
+    },
+  ],
+  locale: 'en',
+  name: 'Joao Tiago',
+  nickname: 'joao.tiago',
+  picture:
+    'https://lh3.googleusercontent.com/a-/AOh14GhcUymAYADTOFTtS7zHdsWbfi2AbCIbpBtxAdNWug',
+  updated_at: '2020-08-21T09:19:39.985Z',
+  user_id: 'google-oauth2|JTsUserId',
+};
+
+const apiUrl = 'https://api.hub.asap.science';
+const invitation_code = 'sampleInvitationCode';
+
+const eventBase = {
+  request: {
+    query: {
+      invitation_code,
+    },
+    body: {},
+  },
+  user,
+  secrets: {
+    ASAP_API_URL: apiUrl,
+    AUTH0_SHARED_SECRET: '',
+    PR_APP_DOMAIN: '',
+    AUTH0_ADDITIONAL_CLAIM_DOMAIN: '',
+  },
+} as DeepPartial<Auth0PostLoginEventWithSecrets> as Auth0PostLoginEventWithSecrets;
+
+const apiBase = {
+  access: {
+    deny: jest.fn(),
+  },
+} as Partial<Auth0PostLoginApi> as Auth0PostLoginApi;
+
```

Comment:
```suggestion
describe('Auth0 Action - Connect User', () => {
```

Reply:
I'd add a root level describe cause otherwise the output looks terrible

Reply:
``` FAIL   test-auth0-rules  apps/auth0-rules/src/__tests__/post-login-connect-user.test.ts
  ● Console

    console.log


      at onExecutePostLogin (src/post-login-connect-user.ts:27:15)

  ● should deny access  an error if fails to connect the user```

Reply:
It has the test filename which just mean it be duplicated information but inconsistent.